training/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Sat Aug 17 12:43:15 ART 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
This was the configuration that was giving me a headache. Don't know why it wasn't set up like this in the first place.